cdap-security/src/main/java/co/cask/cdap/security/store/DummySecureStore.java

```diff
@@ -29,31 +29,37 @@
  * A dummy class that is loaded when the user has set the provider to "kms" but the cluster does not
  * have the required libraries. All operations on this class throw an UnsupportedOperationException.
  */
-public class DummyKMSStore implements SecureStore, SecureStoreManager {
+public class DummySecureStore implements SecureStore, SecureStoreManager {
 
-  private static final String UNSUPPORTED_ERROR_MSG = "Installed version of Hadoop does not support KMS. " +
-    "KMS backed secure store depends on org.apache.hadoop.crypto.key.kms.KMSClientProvider being available. " +
-    "This is supported in Apache Hadoop 2.6.0 and up and on distribution versions that are based " +
-    "on Apache Hadoop 2.6.0 and up.";
+  private static final String SECURE_STORE_SETUP = "Secure store is not configured. To use secure store the provider " +
```

Comment:
this message is really long. It will test the UI if someone ever invokes this. @ajainarayanan :)